gormtestsupport/db_test_suite.go

```diff
@@ -66,7 +66,7 @@ func (s *DBTestSuite) SetupTest() {
 
 // TearDownTest implements suite.TearDownTest
 func (s *DBTestSuite) TearDownTest() {
-	s.clean()
+	// s.clean()
```

Comment:
Hmm

Reply:
well...

Reply:
that's restored in https://github.com/fabric8-services/fabric8-wit/pull/1891/commits/3939d0279318ff9b3f0c602259b742f8a3eae514 (I wanted to load 5000 iterations in my DB with a test to reproduce a `seq_scan`, but I obviously forgot to get rid of that change aftewards..)